src/MagnumPlugins/GltfImporter/GltfImporter.cpp

```diff
@@ -69,8 +69,8 @@
 #include "MagnumPlugins/GltfImporter/decode.h"
 #include "MagnumPlugins/GltfImporter/Gltf.h"
 
-#ifdef CORRADE_MSVC2015_COMPATIBILITY
-/* Otherwise std::unique() fails to compile on MSVC 2015. Other compilers are
+#if defined CORRADE_MSVC2015_COMPATIBILITY || defined CORRADE_TARGET_LIBCXX
```

Comment:
I tried to reproduce with libc++ 14 (latest I have on ArchLinux), but there the compilation passed, so I think it's only happening since version 15 ([this commit in particular](https://github.com/llvm/llvm-project/commit/c9905b8cb0139f410ce63081989a328559e11374), which adds the `static_assert()`). 

So the final change would be the following -- can you confirm I didn't mess up the condition? :)

```suggestion
/* Otherwise std::unique() fails to compile on MSVC 2015 and libc++ 15 (commit
   https://github.com/llvm/llvm-project/commit/c9905b8cb0139f410ce63081989a328559e11374) */
#if defined(CORRADE_MSVC2015_COMPATIBILITY) || (defined(CORRADE_TARGET_LIBCXX) && _LIBCPP_VERSION >= 15) 
```